data/human/human missions.txt

```diff
@@ -5733,3 +5733,228 @@ mission "Argosy Hijacking (Declined)"
+					goto atomic-bombings
+					to display
+						has "Blind Man from Martini: label blind"
+						not "Blind Man from Martini: label atomic-bombings"
+
+				`	"Was your daughter caught in the atomic bombings?"`
+					goto daughter
+					to display
+						or
+							has "Blind Man from Martini: label blind"
+							has "Blind Man from Martini: label illness"
+						not "Blind Man from Martini: label daughter"
+				
+				`	"What happened to your daughter?"`
+					goto illness
+					to display
+						not "Blind Man from Martini: label illness"
+						has "Blind Man from Martini: label daughter"
+
+				`	(Stay silent.)`
+					goto waiting
+					to display
+						not "Blind Man from Martini: label waiting"
+
+			label blind
+			action
+				set "Blind Man from Martini: label blind"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"It was the fourth of July, 3014. I was in my office, on one of the high rises on the outskirts of the capital, with my office window pointed right at the city center." He turns to look out the window, as if he knew New Sydney's skyline was out there where he couldn't see. In the small lull, he takes an audible breath.`
+			`	"It was a beautiful day that day. I... I remember that Pollux was especially bright and colorful that day, and the songbirds that inhabit the city were alive with sound. They were unusually so. It was very vivid, very strange. Maybe I should have listened to the birds...`
+			`	"But when the bomb dropped, I was looking right at it, from my office window. All I could remember next was that I was in a lot of pain, that I couldn't see, but one of my colleagues - thank goodness for her - found me afterwards, during the evacuation, covered in glass from the shockwave. I survived mostly because I was sitting at my terminal and that protected me from the bulk of the shards."`
+				goto taxi
+
+			label blind-1
+			action
+				set "Blind Man from Martini: label blind-1"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. Anything can be taken away from you if you get too used to it."`
+				goto taxi
+			
+			label atomic-bombings
+			action
+				set "Blind Man from Martini: label atomic-bombings"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`
```

Comment:
Are the ` - ` and `- ` spaced differently on purpose?

Reply:
One is used like a semicolon (hopefully correctly), another is a change in the middle of dialogue. I could change the first dash to a semicolon.

Reply:
I think it's fine as-is, just wanted to make sure.